conda_environment_hpc.yml

```diff
@@ -0,0 +1,22 @@
+# Assumes that conda, pip, build-essentials and cuda are installed
+---
+name: ShallowWaterGPU_HPC
+channels:
+- conda-forge
+
+dependencies:
+- python=3.7
+- numpy
+- mpi4py
+- six
+- pytools
+- netcdf4
+- scipy
+
+# Install conda environment (one-time operation):
+# $ conda env create -f conda_environment_hpc.yml
+# Activate environment and install the following packages using pip:
+# $ conda activate ShallowWaterGPU
+#  - pycuda: $ pip3 install --no-deps -U pycuda
```

Comment:
Kan vi legge inn pip som her:
https://stackoverflow.com/questions/35245401/combining-conda-environment-yml-with-pip-requirements-txt

Reply:
Heisann! Jeg gikk bort fra 
```
- pip:
     - pycuda
```
i GPUOcean fordi pip da overkjørte allerede installerte conda-pakker og dermed laga trøbbel med å oppgradere noen pakker der vi trengte ikke-nyeste versjon.

Ser at stackoverflow lister pip også som conda pakke (noe vi ikke har gjort) - er ikke fremmed for at kanskje det ville løst problemet jeg kom over også...